src/components/scheduling/websites/edit.vue

```diff
@@ -96,6 +96,10 @@ export default {
       type: Date,
       default: null,
     },
+    disableDateScheduling: {
```

Comment:
Instead of doing a `disableDateScheduling` prop, I'd recommend using a `dateSchedulingEnabled` prop and set to `true` by default. Then the `v-if` statements don't need to be inverted, i.e. `v-if="dateSchedulingEnabled`